core/capabilities/remote/trigger_publisher.go

```diff
@@ -115,6 +115,11 @@ func (p *triggerPublisher) Receive(_ context.Context, msg *types.MessageBody) {
 		return
 	}
 
+	if msg.ErrorMsg != "" {
+		p.lggr.Errorw("trigger request failed with error",
```

Comment:
nit: change to "received a message with error". At this stage we don't know if it's a trigger request